src/widget.ts

```diff
@@ -143,6 +148,14 @@ export class MapView extends DOMWidgetView {
     this.model.on('change:zoom', this.zoomChanged, this);
     this.model.on('change:center', this.centerChanged, this);
   }
+
+  handleMapClick(event: MapBrowserEvent<MouseEvent>) {
+    const coordinate = event.coordinate;
+    const [lon, lat] = coordinate;
+    this.model.set('clicked_position', [lon, lat]);
+    this.model.save_changes();
```

Comment:
I believe you don't need this in your implementation? You're not using it Python side